addons/mail/models/res_users.py

```diff
@@ -301,11 +307,30 @@ def systray_get_activities(self):
                 "planned_count": 0,
                 "view_type": getattr(Model, '_systray_view', 'list'),
             }
-            for record, activities in activities_by_record.items():
-                if record not in allowed_records:
-                    continue
-                for activity in activities:
-                    user_activities[model_name]["%s_count" % activity.state] += 1
-                    if activity.state in ("today", "overdue"):
-                        user_activities[model_name]["total_count"] += 1
+            if model_name == 'mail.activity':
+                user_activities[model_name]['activity_ids'] = activities.ids
+            for activity in activities:
+                user_activities[model_name]["%s_count" % activity.state] += 1
+                if activity.state in ("today", "overdue"):
+                    user_activities[model_name]["total_count"] += 1
+        if "mail.activity" in user_activities:
+            user_activities["mail.activity"].update({
+                "name": _("Other activities"),
+                "views": [
+                    [self.env.ref('mail.mail_activity_view_tree_readonly').id, "list"],
+                    [self.env.ref('mail.mail_activity_view_form_minimal').id, "form"],
+                ],
+            })
         return list(user_activities.values())
+
+    @api.model
+    def _systray_get_allowed_records(self, res_model, res_ids):
```

Comment:
Maybe
```suggestion
    def _systray_filter_readable_records(self, res_model, res_ids):
```

Reply:
I have integrated in systray_get_activities as the code is very short and we don’t need anymore to have specialized implementation.